frontend/scripts/install_dev_env/install_macos.sh

```diff
@@ -60,7 +60,7 @@ tar -zxv --directory .githooks/. -f ${GOLINT_FILENAME} gitlint
 rm ${GOLINT_FILENAME}
 
 # Change to the frontend directory
-cd frontend
+cd frontend || exit 1
```

Comment:
Is it necessary to have the `|| exit 1`?

Reply:
If you want to avoid IDE and shellcheck warnings, yes.  It's good practice:
https://www.shellcheck.net/wiki/SC2164

Imagine this:
```bash
cd /
....
cd /tmp/junk
rm -rf
```

What if `/tmp/junk` doesn't exist?

Reply:
Got it! Thanks for pointing it out.